integration_tests/features/support/steps.js

```diff
@@ -328,3 +356,41 @@ Then(/Batch transfer of (.*) transactions was a success from (.*) to (.*),(.*)/,
    console.log("All transactions found");
 });
 
+Then(/Transaction status of last result from (.*) to (.*) is known to both wallets/, {timeout: 2*360*10*1000}, async function (walletA, walletB) {
```

Comment:
```suggestion
Then(/transaction status of last result from (.*) to (.*) is known to both wallets/, {timeout: 2*360*10*1000}, async function (walletA, walletB) {
```

Reply:
The name of this method does not really convey what it does. What exactly does it test?

Reply:
It tests if transaction status for the same transaction can be retrieved from both wallets.